guides/common/modules/proc_listing-using-satellite-ansible-modules.adoc

```diff
@@ -1,31 +1,43 @@
 [id="listing-using-satellite-ansible-modules_{context}"]
-== Listing and Using the {Project} Ansible Modules
+== Listing the {Project} Ansible Modules
 
-ifeval::["{build}" == "satellite"]
-You can list the installed Satellite Ansible modules by navigating to the following directory:
+You can list the installed {Project} Ansible modules by navigating to the following directory:
 
+ifeval::["{build}" == "satellite"]
 ----
-$ /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
+# ls /usr/share/ansible/collections/ansible_collections/redhat/satellite/plugins/modules/
 ----
+endif::[]
 
+ifeval::["{build}" != "satellite"]
+----
+# ls /usr/share/ansible/collections/ansible_collections/theforeman/foreman/plugins/modules/
+----
 endif::[]
 
-Alternatively, you can also see the complete list of Satellite Ansible Collection modules and related information on the Automation Hub at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs
+Alternatively, you can also see the complete list of {Project} Ansible modules and other related information at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs in the Automation Hub.
```

Comment:
Upstream, link to the upstream Ansible Galay modules page: https://galaxy.ansible.com/theforeman/foreman

Reply:
Ack.